src/mumble/AudioOutputToken.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+#define MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+
+#include "AudioOutput.h"
+#include "AudioOutputToken.h"
+
+class AudioOutputBuffer;
+
+class AudioOutputToken {
+public:
+	AudioOutputToken() : m_buffer(nullptr){};
+	AudioOutputToken(AudioOutputBuffer *buffer) : m_buffer(buffer){};
+
+	~AudioOutputToken(){};
+
+	inline bool operator==(const AudioOutputToken &rhs) const { return m_buffer == rhs.m_buffer; }
+	inline bool operator!=(const AudioOutputToken &rhs) const { return m_buffer != rhs.m_buffer; }
+
+	explicit operator bool() const { return m_buffer; }
```

Comment:
No, we don't want to make this explicit. Otherwise, we'd have to explicitly use `static_cast` to access this conversion. This conversion, we want to be implicit (just as it is for pointers)
```suggestion
	operator bool() const { return m_buffer; }
```